packaging/docker/build-test.sh

```diff
@@ -46,27 +46,29 @@ do
 	esac
 done
 
-if [ -n "${REPOSITORY}" ] && [ -n "${VERSION}" ] && [ -n "${DOCKER_USERNAME}" ] && [ -n "${DOCKER_PWD}" ] ; then
```

Comment:
This is intentional, so that we can choose the version we want to run.
Unless there is a very specific reason for it, kindly re-instate this (along with the respective references on the messages printed)

Reply:
It is still possible to provide all the variables. I believe all existing functionality is still preserved. This change only makes it all optional so that when doing local development the script will not fail when it attempts to log into Docker.

It doesn't look like this script really adds too much and potentially a simple `docker build` command could suffice for local development. I can revert this whole file if you're prefer to keep all these arguments as mandatory.